### ISSUE TITLE
Typescript definitions for `Polymer.FlattenedNodesObserver` do not have constructor

### ISSUE THREAD

**Issue by eterna2** [Polymer/polymer#5189]
### Description
The typescript definitions for `Polymer.FlattenedNodesObserver` for Polymer do not have a constructor defined when arguments are expected. This cause typescript to incorrectly flag the error `expected 0 arguments but got 2 .` when creating a new instance of `FlattenedNodesObserver`.

#### Steps to Reproduce
1. Create a instance of  `FlattenedNodesObserver`
```ts
this._observer = new Polymer.FlattenedNodesObserver(this, (info: any) => {
   this.info = info;
});
```

#### Expected Results
No error flagged.

#### Actual Results
`expected 0 arguments but got 2 .`  error flagged.

### Versions
- Polymer: v2.6.0
- Typescript: 2.8.1

**Comment by TimvdLippe** [comment]
This is pending a release of https://github.com/Polymer/gen-typescript-declarations @aomarks could you make a release for that? Then I can make the update here in this repo.

**Comment by aomarks** [comment]
> This is pending a release of https://github.com/Polymer/gen-typescript-declarations @aomarks could you make a release for that? Then I can make the update here in this repo.

Published @polymer/gen-typescript-declarations@1.3.0

**Comment by TimvdLippe** [comment]
Fixed in #5281
